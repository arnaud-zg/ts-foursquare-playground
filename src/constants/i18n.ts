export const i18n = {
  ACTION_EXAMPLE: 'Ex:',
  ACTION_NO_NEED_PAYLAOD: "Current action doesn't need any payload",
  ACTION: 'Action',
  CANCEL: 'Cancel',
  CREDENTIALS_PLACEHOLDER: '0123456789',
  CREDENTIALS: 'Credentials',
  EXAMPLE: 'Ex:',
  FAILURE: 'Failure',
  HEADER_TITLE: '"ts-foursquare" is just a simple library',
  PAYLOAD: 'Payload',
  REQUEST: 'Request',
  SEND: 'Send',
  SUCCESS: 'Success',
}
