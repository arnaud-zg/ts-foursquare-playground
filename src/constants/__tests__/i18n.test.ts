import { i18n } from '../i18n'

describe('i18n', () => {
  it('should make a snapshot', () => {
    expect(i18n).toMatchSnapshot()
  })
})
