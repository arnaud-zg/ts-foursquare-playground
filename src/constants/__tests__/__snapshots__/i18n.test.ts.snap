// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`i18n should make a snapshot 1`] = `
Object {
  "ACTION": "Action",
  "ACTION_EXAMPLE": "Ex:",
  "ACTION_IS_CANCELLED_BY_USER": "Action is cancelled by user",
  "ACTION_NO_NEED_PAYLAOD": "Current action doesn't need any payload",
  "CANCEL": "Cancel",
  "CREDENTIALS": "Credentials",
  "CREDENTIALS_PLACEHOLDER": "0123456789",
  "EXAMPLE": "Example",
  "FAILURE": "Failure",
  "HEADER_TITLE": "\\"ts-foursquare\\" is just a simple library",
  "INVALID_INTENT": "Invalid intent",
  "PAYLOAD": "Payload",
  "REQUEST": "Request",
  "RESET": "Reset",
  "SEND": "Send",
  "SUCCESS": "Success",
}
`;
